wpapi-class-content.php

```diff
@@ -39,11 +42,21 @@ function get_pages($wp_api_posts){
     function get_media($wp_api_posts, $size){
         $html = '';
         foreach ($wp_api_posts as $k => $v){
-            $id = $v->ID;
-            $title = $v->title;
-            $imgsrc = $v->attachment_meta->sizes->$size->url;
-            $html .= "<li><a href='{$imgsrc}'><img src='{$imgsrc}'><h2 class='wpapi-title'>{$title}</h2></a></li>";
+			if ( ! isset( $v['media_details']['sizes'] ) || ! $v['media_details']['sizes'] ) {
+				continue;
+			}
+            $id = $v['id'];
+            $title = $v['title']['rendered'];
+			$img = $v['media_details']['sizes'];
+			$link = $v['link'];
+			$alt = $v['alt_text'];
+			if ( isset( $img[ $size ] ) && $img[ $size ] ) {

```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found